### ISSUE TITLE
Fail to be triggeredd with jQuery 1.8+

### ISSUE THREAD

**Issue by pentie** [eisbehr-/jquery.lazy#45]
in the setup step:
```Javascript
        // set up lazy
        (function()
        {
            // if event driven don't wait for page loading
            if( configuration("bind") == "event" ) _initialize();

            // otherwise load initial items and start lazy after page load
            else $(window).load(_initialize);
        })();
```

the use of `$().load()` event handler is depreciated since jQuery 1.8: http://api.jquery.com/load-event/

shoud be changed to `.on( "load", _initialize)`

**Comment by eisbehr-** [comment]
Hello @pentie 
Thanks for your feedback. I will change it within the next version.

**Comment by eisbehr-** [comment]
Fixed @[1.6.6](https://github.com/eisbehr-/jquery.lazy/releases/tag/1.6.6)
